src/mgmt/osp-sah.ks

```diff
@@ -596,14 +593,13 @@ fi
 subscription-manager repos ${ProxyInfo} '--disable=*' --enable=rhel-7-server-rpms --enable=rhel-7-server-optional-rpms
 
 systemctl disable NetworkManager
-systemctl disable firewalld
 systemctl disable chronyd
```

Comment:
Shouldn't this be removed since we need chronyd enabled?

Reply:
Yes , updated.